src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -46,25 +46,14 @@ public void storeMessage(Message message) {
     datastore.put(messageEntity);
   }
 
-  /**
-   * Gets messages posted by a specific user.
-   *
-   * @return a list of messages posted by the user, or empty list if user has never posted a
-   *     message. List is sorted by time descending.
-   */
-  public List<Message> getMessages(String user) {
+  public List<Message> helper(PreparedQuery results) {
```

Comment:
It's a really good idea to refactor the code like this into a helper function. I'm wondering if it can have a more descriptive name. Since it converts a list of query results into entities, maybe we can call it message converter? I'm open to other names you can come up with (I'm terrible at naming things...)